rdt-app/src/main/java/io/ona/rdt_app/activity/RDTJsonFormActivity.java

```diff
@@ -43,35 +37,31 @@ private void modifyActionBarAppearance() {
 
     @Override
     public void initializeFormFragment() {
-        this.isFormFragmentInitialized = true;
         RDTJsonFormFragment jsonFormFragment = (RDTJsonFormFragment) RDTJsonFormFragment.getFormFragment(JsonFormConstants.FIRST_STEP_NAME);
         getSupportFragmentManager().beginTransaction().add(com.vijay.jsonwizard.R.id.container, jsonFormFragment).commitAllowingStateLoss();
     }
 
     @Override
     public void onRequestPermissionsResult(int requestCode, @NonNull String[] permissions, @NonNull int[] grantResults){
-        if (requestCode == REQUEST_CAMERA_PERMISSION && grantResults.length > 0) {
-            if (grantResults[0] == PackageManager.PERMISSION_GRANTED && ActivityCompat.checkSelfPermission(this, Manifest.permission.READ_PHONE_STATE) != PackageManager.PERMISSION_GRANTED) {
-                this.propertyManager.grantPhoneStatePermission();
-            } else if (grantResults[0] == PackageManager.PERMISSION_DENIED) {
-                Intent resultIntent = new Intent();
-                setResult(RESULT_CANCELED, resultIntent);
-                formUtils.showToast(this, "RDT image capture requires camera permission");
-                finish();
-            }
-        } else if (requestCode == PHONE_STATE_PERMISSION && grantResults.length > 0) {
-            if (grantResults[0] == PackageManager.PERMISSION_GRANTED && !isFormFragmentInitialized) {
-                initializeFormFragment();
-            } else if (grantResults[0] == PackageManager.PERMISSION_DENIED) {
-                formUtils.showToast(this, "Phone state permissions are required to submit this form");
-                finish();
-            }
-        }
-        super.onRequestPermissionsResult(requestCode, permissions, grantResults);
+       if (requestCode == PHONE_STATE_PERMISSION && grantResults.length > 0 && grantResults[0] == PackageManager.PERMISSION_DENIED) {
```

Comment:
Are Camera permissions not required any more

Reply:
These are phone state permissions.

Just providing a custom way of reacting to someone refusing to grant them by closing the form.

Reply:
It's not clear that we need a complex way to handle this for now.

Created an issue to track this https://github.com/onaio/rdt-standard/issues/92